test1-prod.js

```diff
@@ -3,3 +3,10 @@ var fooProd = '' ;
 if(fooProd  === "bar") {
   console.log( 'hey' )
 }
+
+
+var fooProd2 = '' ;
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/machadoit-test-org/eslint-test-prod/pullRequest?prid=8631734)
```suggestion
var fooProd2 = "" ;
```